infinoted/infinoted-parameter.h

```diff
@@ -240,6 +244,11 @@ infinoted_parameter_convert_flags(gpointer out,
                                   const GFlagsValue* values,
                                   GError** error);
 
+gboolean
+infinoted_parameter_convert_ip_address(gpointer out,
+                                       gpointer in,
+                                       GError** error);
+
```

Comment:
For this function to show up in the documentation, it would need to be added to `docs/reference/libinfinoted-plugin-manager/libinfinoted-plugin-manager-0.7-sections.txt`.